frontend/src/components/auth/PrivateRoute.tsx

```diff
@@ -18,7 +18,10 @@ const PrivateRoute: React.FC<PrivateRouteProps> = ({
   const { authenticatedUser } = useContext(AuthContext);
 
   return authenticatedUser ? (
-    <Route path={path} exact={exact} component={component} />
+    <>
+      <Route path={path} exact={exact} component={component} />
+      <Redirect to={CAMPS_PAGE} />
```

Comment:
Why do we need this Redirect here? If a target `path` is being passed then I think we don't need to redirect the user to the camps page again since we want them to go to the specified path?